### PR TITLE
Do not fail server on exception without arguments

### PR DESCRIPTION
**Pull request by AleksandrIakhnev**

### CODE REVIEW

File: execution.py [hunk 1]
```diff
@@ -94,7 +95,13 @@ def execute(self, execution_context, results):
...
                                   exc_info=1)
                 stop_test = 'stoptest' in type(error).__name__.lower()
-                results.failed(instruction, error.args[0], stop_test)
+
+                if len(error.args) > 0:
+                    cause = error.args[0]
+                else:
+                    cause = '\n'.join([str(type(error)),traceback.format_exc()])
+
+                results.failed(instruction, cause, stop_test)
```
Comment (mikeplavsky) [inline execution.py:104]: It still fails, does not it?

Comment (AleksandrIakhnev) [reply execution.py:104]: It shows exception as in a normal case for exception and continues test running. Without fix tests are stopped if someone make something like **'raise Exception()'**

Comment (mikeplavsky) [reply execution.py:104]: I see, you stopped using args[0]

**Comment by mikeplavsky** [comment]
LGTM 

P.S. What about tests?

**Comment by AleksandrIakhnev** [comment]
Plan to add tests, configure travis for this repository at least. Probably we can run even Fitnesse tests in travis because it supports containers-based environment.

**Pull request merged**